cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala

```diff
@@ -214,7 +214,7 @@ object SparkMainWrapper {
           executor.awaitTermination(5L, TimeUnit.SECONDS)
           // Send the complete call
           client.completed(workflowToken)
-          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun)
+          LOG.info("Spark program {} completed", programRunId.getProgram)
```

Comment:
Removing the '.' after the error and the Run ID which is included in the "started" message. Also fixing the build error.